diff --git a/src/ImageData/LabelImageData.php b/src/ImageData/LabelImageData.php
index ccc89de..bd265af 100644
--- a/src/ImageData/LabelImageData.php
+++ b/src/ImageData/LabelImageData.php
@@ -16,10 +16,6 @@ final readonly class LabelImageData
 
     public static function createForLabel(LabelInterface $label): self
     {
-        if (str_contains($label->getText(), "\n")) {
-            throw new \Exception('Label does not support line breaks');
-        }
-
         if (!function_exists('imagettfbbox')) {
             throw new \Exception('Function "imagettfbbox" does not exist: check your FreeType installation');
         }
